### DISCUSSION TITLE
Cleaning up your WSUS Server

### DISCUSSION THREAD

**Discussion by Umps238** [alkampfergit/personal-blog#9] [Q&A]
Hi, Ive come across your article posted back in Aug 2015.  Ive run through the steps however, i have machines still reporting needing driver updates ive now deleted, how do i clean this up so the machines no longer report rouge updates?  When checking what updates they require it will not show them as they are not available.

Thanks

**Comment by alkampfergit** [comment]
Sorry for the late answer, honestly my WSUS server was dimissed last year, so I have no way to reproduce your issue. :( Honestly it didn't happen to me, so I have little idea of what could be the reason for your problem.
